src/services/UserService.ts

```diff
@@ -118,7 +122,10 @@ const getInitializedTimeZone = (user: User, timeZone: string): Promise<Initializ
   if (cachedTimeZone.promise) {
     // clear the promise if this was cached before user acknowledged the time zone
     const { timeZoneAcknowledgedOnMs } = CachedUserService.getUserPreferences();
-    if (!isNaN(timeZoneAcknowledgedOnMs) && timeZoneAcknowledgedOnMs > cachedTimeZone.cachedOn) {
+    if (
+      (!isNaN(timeZoneAcknowledgedOnMs) && timeZoneAcknowledgedOnMs > cachedTimeZone.cachedOn) ||
+      cachedTimeZone.updated
```

Comment:
this brings back a potential double setting of time zone.. no point in having this cached promise..
ugh this feature is getting messy or we need to rethink how we do this..

Reply:
might as well remove this cached promise.. maybe create a ticket to handle double setting of user time zone or something like that.. for later

Reply:
Yes :(

Reply:
But at least when timezone is already set and updated is false we are not setting time zone again

Reply:
the user and org notifications are initialized at the same time and both will race to set the time zone.
i'm very sad about this change.. like why even make this change when we know it's pointless.
might as well remove it

Reply:
another idea.. if you think it helps.. we can make the org notification take in the user time zone as a prop
and until that is valid.. the org notification renders nothing?

Reply:
> I mean that only with a new user the "updated" will be true, so the timezone will be initialized two times. If user has the timezone already set, it will execute it just one time, org will use the cached one (because updated will be false)

Got it.. but the two times is also not desirable :) 
This is definitely better than the scrolling issue.. so let's get this in. We can discuss offline later if we want to clean this up some more.